src/main/resources/db_scripts/createDB.sql

```diff
@@ -0,0 +1,63 @@
+CREATE DATABASE quizowanie;
+USE quizowanie;
+
+CREATE TABLE question (
+	quest_id INT NOT NULL,
```

Comment:
Przecież widać do czego to id jest, zawsze występuje z nazwą tabelki, a przy JOIN masz krótszy zapis itd.
Mam na myśli, że to pole można nazwać `question_id` lub (dla mnie lepiej) `id`